monkey/tests/unit_tests/monkey_island/cc/services/agent_plugin_service/test_mongo_agent_plugin_repository.py

```diff
@@ -240,6 +240,16 @@ def test_store_agent_plugin(agent_plugin_repository: MongoAgentPluginRepository)
     assert plugin == FAKE_AGENT_PLUGIN_1
 
 
+def test_store_agent_plugin__excludes_source_archive(
+    mongo_client, agent_plugin_repository: MongoAgentPluginRepository
+):
+    agent_plugin_repository.store_agent_plugin(OperatingSystem.LINUX, FAKE_AGENT_PLUGIN_1)
+
+    assert mongo_client.monkey_island.agent_plugins.count_documents({}) == 1
+    plugin_dict = mongo_client.monkey_island.agent_plugins.find_one({})
+    assert plugin_dict.get("source_archive") is None
```

Comment:
I think the meaning is clearer:
```suggestion
    assert "source_archive" not in plugin_dict
```